data/data-transfer/export_data_database/spec.yaml

```diff
@@ -0,0 +1,13 @@
+$schema: http://azureml/sdk-2-0/DataTransferComponent.json
+name: export_data_database
+display_name: export data from uri_file data asset to database
```

Comment:
https://github.com/Azure/azureml-assets/blob/main/responsibleai/tabular/components/counterfactual/spec.yaml#L4

suggest to provide learn more links